gmeta/Cargo.toml

```diff
@@ -1,13 +1,19 @@
 [package]
 name = "gmeta"
-version.workspace = true
+version = "0.1.0"
+description = "Gear metadata"
+authors.workspace = true
 edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+documentation.workspace = true
 
 [dependencies]
 scale-info.workspace = true
 blake2-rfc.workspace = true
 hex = { workspace = true, features = ["alloc"] }
-gmeta-codegen = { path = "codegen", optional = true }
+gmeta-codegen = { version = "0.1.0", path = "codegen", optional = true }
```

Comment:
why do we need version specification here?

Reply:
this is tricky 

> However, crates that use dependencies specified with only a path are not permitted on [crates.io](https://crates.io/).

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies